Formula/catboost.rb

```diff
@@ -0,0 +1,24 @@
+class Catboost < Formula
+  desc "Fast, scalable, high performance Gradient Boosting on Decision Trees library"
+  homepage "https://catboost.ai"
+  url "https://github.com/catboost/catboost/archive/v0.24.4.tar.gz"
+  sha256 "03df498249206519b7e37ff8067cf265a2826afc4305a32f97986fbe308994da"
+  license "Apache-2.0"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  def install
+    cd "#{buildpath}/catboost/app" do
+      ENV["YA_CACHE_DIR"] = "./.ya"
+      system "../../ya", "make", "-r", "-o", "#{buildpath}/build"
+      bin.install "#{buildpath}/build/catboost/app/catboost"
+    end
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/catboost --version")
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula